tasks/src/document-question-answering/about.md

```diff
@@ -1,18 +1,22 @@
+Document Question Answering models can be used to answer natural language questions about documents. Typically, document QA models consider textual, layout and potentially visual information. This is useful when the question requires some understanding of the visual aspects of the document.
+Nevertheless, certain document QA models can work without document images. Hence the task is not limited to visually-rich documents and allows to ask questions on spreadsheets, text PDFs, etc!
 
 ### Document Parsing
 
-You can use Document question answering models to parse forms. For example, you can extract the name, address, and other information from a form. You can also use the model to extract information from a table, or even a resume.
+One of the most popular use cases of document question answering models is the parsing of structured documents. For example, you can extract the name, address, and other information from a form. You can also use the model to extract information from a table, or even a resume.
 
 ### Invoice Information Extraction
 
-You can use Document question answering models to extract information from invoices. For example, you can extract the invoice number, the invoice date, the total amount, the VAT number, and the invoice recipient.
+Another very popular use case is invoice information extraction. For example, you can extract the invoice number, the invoice date, the total amount, the VAT number, and the invoice recipient.
 
 ## Inference
 
-You can infer with Document QA models with the 🤗 Transformers library using the [`document-question-answering` pipeline](https://huggingface.co/docs/transformers/en/main_classes/pipelines#transformers.DocumentQuestionAnsweringPipeline). If no model checkpoint is given, the pipeline will be initialized with [`impira/layoutlm-document-qa`](https://huggingface.co/impira/layoutlm-document-qa). This pipeline answers the question(s) given as inputs by using the document(s). A document is defined as an image and an optional list of (word, box) tuples which represent the text in the document. If the word_boxes are not provided, it will use the Tesseract OCR engine (if available) to extract the words and boxes automatically for LayoutLM-like models which require them as input. For Donut, no OCR is run (see [paper](https://arxiv.org/abs/2111.15664)).
+You can infer with Document QA models with the 🤗 Transformers library using the [`document-question-answering` pipeline](https://huggingface.co/docs/transformers/en/main_classes/pipelines#transformers.DocumentQuestionAnsweringPipeline). If no model checkpoint is given, the pipeline will be initialized with [`impira/layoutlm-document-qa`](https://huggingface.co/impira/layoutlm-document-qa). This pipeline takes question(s) and document(s) as input. 
+In this case, we define a document as an image and an optional list of (word, box) tuples representing the text in the document. The pipeline will then return the answer to the question in the document
```

Comment:
I don't quite understand what you mean by 

> "an optional list of (word, box) tuples"

Could you clarify?

Reply:
@KayO-GH this definition is taken from the Document QA pipeline [documentation](https://huggingface.co/docs/transformers/main_classes/pipelines#transformers.DocumentQuestionAnsweringPipeline).

Some document qa models (e.g. LayoutLM family) expect a list of words and associated bounding boxes as input. This is usually the output of the OCR (supported by the pipeline), but they can **optionally** be provided in the pipeline args (for example, when you already have the output of a third-party OCR).
I agree that this could be clearer, and might be a little bit cryptic to a person seeing these for the first time. Will try to simplify 😊